titus-server-master/src/test/java/com/netflix/titus/master/kubernetes/pod/v0/V0SpecPodFactoryTest.java

```diff
@@ -1,338 +0,0 @@
-/*
```

Comment:
Two tests here that are being deleted that do not have equivilants in v1specpodfactorytest

relocationLabel
testCapacityGroupAssignment

If you think it is worth it, please copy these over